app/components/organization/Pipeline/bar.js

```diff
@@ -69,7 +69,11 @@ class Bar extends React.Component {
 
     if (this.props.href) {
       return (
-        <AnchoredPopover position="absolute" style={{ height: '100%', left: this.props.left, bottom: 0 }}>
+        <AnchoredPopover
+          position="absolute"
+          style={{ height: '100%', left: this.props.left, bottom: 0 }}
+          width={300}
```

Comment:
Actually, it should really be hard coded in these two places, because it's very much dependent on the size of the content inside the AnchoredPopover. Perhaps the AnchoredPopover shouldn't have a default size at all?